### PR TITLE
Bugfix issue 89

### PR DESCRIPTION
**Pull request by sgettys**
Sanity check

### CODE REVIEW

File: pkg/terraform/install.go [hunk 1]
```diff
@@ -57,10 +57,13 @@ func (m *Mixin) Install() error {
 			fmt.Fprintf(m.Err, "DEBUG: TF var file created:\n%s\n", string(vbs))
 		}
 	}
-	for _, k := range sortKeys(step.Vars) {
```
Comment (sgettys) [inline pkg/terraform/install.go:60]: We're losing this key sorting, not sure the benefit of it but might want to revert back to this, thoughts?

Comment (bdegeeter) [reply pkg/terraform/install.go:60]: can you use a modifier to sort and get the same behavior?  https://github.com/tidwall/gjson/blob/v1.14.2/SYNTAX.md#modifiers

Comment (sgettys) [reply pkg/terraform/install.go:60]: Yeah looking at that. Also might want to do the same thing with backend config

File: scripts/test/test-cli.sh [hunk 2]
```diff
@@ -33,17 +33,25 @@ cp -r ${REPO_DIR}/build/testdata/bundles/terraform/terraform .
 cp ${REPO_DIR}/build/testdata/bundles/terraform/porter.yaml .
 
 ${PORTER_HOME}/porter build
-
-${PORTER_HOME}/porter install --debug --param file_contents='foo!'
+#terraform plan --var file_contents="foo!" --var map_var='{"foo": "bar"}' --var array_var='["foo", "bar"]'
```
Comment (jsevedge) [inline scripts/test/test-cli.sh]: nit: remove commented out code

**Pull request merged**